apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/ReadTests.cs

```diff
@@ -591,5 +591,28 @@ public async Task GetOrdinalAsync()
                 }
             }
         }
+
+        [Fact]
+        public async Task WithCommand()
+        {
+            using (var connection = _fixture.GetConnection())
+            {
+                // Create a generic command and let the type be determined based on the command text.
+                // A statement that starts with a WITH clause should be treated as a query.
+                var cmd = connection.CreateCommand();
```

Comment:
Not sure why GitHub isn't showing me a suggestion option, but assuming Spanner doesn't mind line breaks, this can be simplified to:

                cmd.CommandText = $@"
                     WITH LastValue AS (
                        SELECT Key, StringValue
                        FROM {_fixture.TableName}
                        ORDER BY StringValue
                        LIMIT {int.MaxValue} OFFSET {_fixture.RowCount - 1})
                      SELECT * FROM LastValue";